.github/workflows/ci.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
           toolchain: ${{ matrix.rust }}
           components: clippy
       - uses: r7kamura/rust-problem-matchers@v1
-      - run: cargo +${{ matrix.rust }} clippy --all --all-features -- -D warnings
+      - run: cargo +${{ matrix.rust }} clippy --all --all-features -- -D warnings `${{ matrix.rust == 'beta' }} && echo "-Aclippy::incorrect_clone_impl_on_copy_type"`
```

Comment:
A little hackier than I like, but you can see the failed attempt of me trying to suppress this targeted in the lib.rs file at https://github.com/mobilecoinfoundation/sgx/actions/runs/5798566937.

clippy will error if you try to suppress a check it doesn't know about. Which means you can't suppress a `beta` check during a `stable` run :(.